docs/source/api.rst

```diff
@@ -184,6 +308,174 @@ HVAC
+Property                           Type     Units                       Constraints     Required  Default             Notes
+=================================  =======  ==========================  ==============  ========  ==================  ==============================================       
+``id``                             id                                   Must be unique  yes       PSC    
+``systemType``                     string                                               yes       
+``conditionedFloorAreaServed``     float    ft2                         >0              no        PSC
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+
+
+HVAC Control Systems
+~~~~~~~~~~~~~~~~~~~~
+
+If any HVAC systems are specified, a single control system can be entered in ``...building.systems.hvac.hvacControlSystems``.
+
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+Property                           Type     Units                       Constraints     Required  Default             Notes
+=================================  =======  ==========================  ==============  ========  ==================  ==============================================       
+``id``                             id                                   Must be unique  yes       HVACControl1
+``heatingSeason``                  object                                                                             Thermostat settings for heating season
+``coolingSeason``                  object                                                                             Thermostat settings for cooling season
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+
+``heatingSeason`` and ``coolingSeason`` objects share the following schema:
+
+=================================  =======  ==========================  ==============  ========  ========================  ============================================== 
+Property                           Type     Units                       Constraints     Required  Default                   Notes
+=================================  =======  ==========================  ==============  ========  ========================  ==============================================       
+``setpointTemp``                   float    F                                           no        Heating: 67, Cooling: 78  
+``setbackTemp``                    float    F                                           no        null                      Also referred to as setup temperature for the cooling season.
```

Comment:
I've seen both versions with and without the `Temp` suffix for both properties in different places. Which option should we keep around? `setpointTemp`, `setbackTemp` or `setpoint`, `setback`?

Reply:
let's stick with `Temp`.

Reply:
@kellyatradiantlabs I've just read your comment! Oh my god, how I missed this one. I asked that question because I wanted to align our code with the specs and docs here. So, we'll make the change soon in the code and everywhere to include that suffix, `Temp`.